LICENSE

```diff
@@ -1,4 +1,21 @@
-File containning the license of your package.
+The MIT License (MIT)
```

Comment:
This License file is the license of the package, not the Software (wich is in the manifest). So, no, it's not owned by the Gitlab e.V company.

Reply:
Ok, I missunderstood something :-) I can change the license, keep MIT but without Gitlab, or let the owner of the repo set by himself.